### DISCUSSION TITLE
Mega Menu : Exit animation for Dropdown Effect

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21881] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi

Currently, there is only Animation option to display mega menu content. But the downside is that the animation option only sets the input animation for mega menu content. Therefore, it has no control over the output animation of mega menu content.

### Describe the solution you'd like

Please add a new option to control the output animation of the megamenu content so that we can control the output animation in the mega menu content in addition to the current option.
Just like the popup, which has the possibility to set the **entrance** and **exit** animation.

**Comment by macdeesh** [comment]
Hi @Michaelgimii, did you find a solution for the exit animation ?? Have you used the method here (https://developers.elementor.com/docs/editor-controls/control-exit-animation/) ?
